cmd/utils/flags.go

```diff
@@ -157,6 +157,11 @@ var (
 		Usage: "Document Root for HTTPClient file scheme",
 		Value: DirectoryString{homeDir()},
 	}
+	ExitWhenSyncedFlag = cli.DurationFlag{
+		Name:  "exitwhensynced",
+		Usage: "Exists syncing by given time (default 0) after block synchronisation",
+		Value: -1,
+	}
```

Comment:
I could understand this being a boolean flag, but what's the purpose of configurable time value for the wait interval. Imho `--exitwhensync` is obvious, whereas `--exitwhensync=13` is weird.

Reply:
Hi, This was specifically requested in the issue, number 16400.  It was specifically requested that this flag have an argument N, with N being the amount of time to keep running once the chain is in sync. N=0 means geth exists immediately once the chain is caught up.  How would you be interested in proceeding?

Reply:
I can change name to something more accurate or would you prefer something else

Reply:
any thoughts?

Reply:
Do we just want to make it a boolean or change it to something like timetoexitwhensynced?